Makefile

```diff
@@ -60,7 +61,7 @@ run: server
 	./$(APP_NAME)
 
 test: 
-	godep go test ./...
+	godep go test $(GOFILES)
```

Comment:
godep go test $(GOFILES), would this run all the testfiles in all the vendor packages too ?